cmake/partition_manager.cmake

```diff
@@ -11,22 +11,35 @@ if(FIRST_BOILERPLATE_EXECUTION)
     PARTITION_MANAGER_CONFIG_FILES
     )
 
-  if(partition_manager_config_files)
+  set(partitions_yml ${APPLICATION_SOURCE_DIR}/prj.partitions.yml)
+
+  if(partition_manager_config_files OR (EXISTS ${partitions_yml}))
     # Partition manager is enabled because we have populated config
-    # files.
+    # files or because the application has pointet to pre defined partition
+    # configurations.
 
-    execute_process(
-      COMMAND
-      ${PYTHON_EXECUTABLE}
+    if (EXISTS ${partitions_yml})
+      set(pre_defined_conf --pre-defined-config ${partitions_yml})
```

Comment:
we should consistently use pre-defined or predefined. The docs use predefined.

Reply:
will be `static`